mkdocs_simple_hooks/__init__.py

```diff
@@ -39,9 +40,12 @@ def handle_hook(self, *args, hook, **kwargs):
         return hook(*args, **kwargs)
 
     def _get_function(self, hook_path, warns):
-        cwd = os.getcwd()
-        if cwd not in sys.path:  # pragma: no cover
-            sys.path.append(cwd)
+        cwd = Path(os.getcwd())
+        root_dir = cwd.parent
+        if str(cwd) not in sys.path or str(root_dir) not in sys.path:  # pragma: no cover
```

Comment:
These should be two separate conditions. If `cwd` not in path - append only `cwd`. If `root_dir` not in path - append only `root_dir`.

Reply:
I separated the conditions in commit [fc18](https://github.com/aklajnert/mkdocs-simple-hooks/pull/7/commits/fc18c56c290d0205cb56dec8b7df64e9566b1c91).